pkg/sqlite/query.go

```diff
@@ -1117,3 +1117,90 @@ func (s *SQLQuerier) GetPropertiesForBundle(ctx context.Context, name, version,
 
 	return
 }
+
+func (s *SQLQuerier) GetTailFromBundle(ctx context.Context, name string) (bundles []string, err error) {
```

Comment:
The querier interface is generally to define the interface of the grpc API into requests against the database. If we aren't exposing this externally, shouldn't we put this in `loadprocs.go` or where ever it is actually used? It feels like we are abusing this interface this way.